react-native-in-app-payments-quickstart/package.json

```diff
@@ -5,43 +5,41 @@
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
-    "start": "react-native start",
-    "test": "jest",
     "lint": "eslint .",
-    "build:ios": "react-native bundle --entry-file='index.js' --bundle-output='./ios/main.jsbundle' --dev=false --platform='iOS'"
```

Comment:
I'll trust you, but surprised that the ios build line vanished here.  Not my swim lane, however; I assume it's somewhere else now!